diff --git a/assets/localization/en.json b/assets/localization/en.json
index 25e15175..df4f5297 100644
--- a/assets/localization/en.json
+++ b/assets/localization/en.json
@@ -27,7 +27,7 @@
   "%general_run%": "Run",
   "%mainMenu_help%": "Help",
   "%mainMenu_layout%": "Layout",
-  "%mainMenu_about%": "About Platform.Bible",
+  "%mainMenu_about%": "About {{ productInfo.productName }}",
   "%mainMenu_downloadInstallResources%": "Download/Install Resources",
   "%mainMenu_exit%": "Exit",
   "%mainMenu_openDeveloperDocumentation%": "Open Developer Documentation",
@@ -37,7 +37,7 @@
   "%mainMenu_project%": "Project",
   "%mainMenu_window%": "Window",
   "%menuItemName_wordList%": "Word List",
-  "%product_name%": "Platform.Bible",
+  "%product_name%": "{{ productInfo.productName }}",
   "%project_full_name_missing%": "*Name Missing*",
   "%project_language_missing%": "*Language Missing*",
   "%project_name_missing%": "_NoName_",
diff --git a/c-sharp/Projects/LocalParatextProjects.cs b/c-sharp/Projects/LocalParatextProjects.cs
index 01c5acee..e0d9e68f 100644
--- a/c-sharp/Projects/LocalParatextProjects.cs
+++ b/c-sharp/Projects/LocalParatextProjects.cs
@@ -49,7 +49,7 @@ internal class LocalParatextProjects
     {
         ProjectRootFolder = Path.Join(
             Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
-            ".platform.bible",
+            ".{{ productInfo.name }}",
             "projects",
             "Paratext 9 Projects"
         );
diff --git a/src/node/utils/util.ts b/src/node/utils/util.ts
index 7b6b97cd..7f43954b 100644
--- a/src/node/utils/util.ts
+++ b/src/node/utils/util.ts
@@ -38,7 +38,7 @@ export function resolveHtmlPath(htmlFileName: string) {
  */
 export const getAppDir = memoizeOne((): string => {
   return globalThis.isPackaged
-    ? path.join(os.homedir(), '/.platform.bible')
+    ? path.join(os.homedir(), '/.{{ productInfo.name }}')
     : path.join(__dirname, '../../../dev-appdata');
 });
 
diff --git a/src/renderer/index.ejs b/src/renderer/index.ejs
index bba6b888..830b3f57 100644
--- a/src/renderer/index.ejs
+++ b/src/renderer/index.ejs
@@ -108,7 +108,7 @@
         form-action 'self';
         navigate-to 'none';"
     />
-    <title>Platform.Bible</title>
+    <title>{{ productInfo.productName }}</title>
   </head>
   <body>
     <div id="root"></div>
